docs/tutorials/slurm/compute_canada/compute_canada.rst

```diff
@@ -36,49 +37,72 @@ Log in to the Compute Canada server you copied your data to:
 Setting up and using MIRACL
 ===========================
 
-Load the specific Singularity module you would like to use (e.g. Singularity 3.5):
+Load the specific Apptainer/Singularity module you would like to use (e.g. Apptainer 1.2.4):
 
 .. code-block::
 
-   $ module load singularity/3.5
+   $ module load apptainer/1.2.4
 
 .. note::
    Not specifying the version will load the latest version available on your node
 
 Since :program:`MIRACL` will take up a significant amount of space, it is 
 recommended to download and work with the :program:`MIRACL` 
-:program:`Singularity` container in the scratch directory. First, navigate 
+:program:`Apptainer/Singularity` container in the scratch directory. First, navigate 
 there:
 
 .. code-block::
    
    $ cd $SCRATCH
 
-Then pull (download) the :program:`Singularity` container:
+Then pull (download) the :program:`Apptainer/Singularity` container:
 
 .. code-block::
 
-   $ singularity pull miracl_latest.sif library://aiconslab/miracl/miracl:latest
+   $ apptainer pull miracl_latest.sif library://aiconslab/miracl/miracl:latest
+
+.. attention::  
+
+   If you encouter the following error with the pull command:  
+
+   .. code-block:: 
+   
+      FATAL:   Unable to get library client configuration: remote has no library client (see https://apptainer.org/docs/user/latest/endpoint.html#no-default-remote)  
+
+   This error can be resolved by adding the default Apptainer end point from which the container is downloaded. Use:  
+
+   .. code-block::  
+      apptainer remote add --no-login SylabsCloud cloud.sylabs.io  
+
+   then:  
+
+   .. code-block::  
```

Comment:
Same thing here. Make sure there is a separating, empty line between the directive and its content.